c2cgeoportal/scripts/c2ctool.py

```diff
@@ -226,9 +235,9 @@ def test_checkers(self):
             if resp.status < 200 or resp.status >= 300:
                 return False, "\n".join([
                     "Checker error:",
-                    "Open `http://{}{}{}` for more informations."
+                    "Open `{}` for more information."
                 ]).format(
-                    self.project["host"], self.project["checker_path"], check_type
+                    checker_url
```

Comment:
If host is not localhost this message becomes wrong, if you want to change it you should do something like in 2.3, with the curl command...

Reply:
OK, new commit coming